PolyEngine/Engine/Src/Input/InputSystem.cpp

```diff
@@ -51,7 +56,7 @@ void InputSystem::InputPhase(Scene* world)
 			com->MouseDelta = ev.Pos;
 			break;
 		case eInputEventType::MOUSEPOS:
-			com->MouseDelta = ev.Pos - com->MousePos;
+			// com->MouseDelta = ev.Pos - com->MousePos;
```

Comment:
why?

Reply:
Suddenly MousePos i overwriting com->MouseDelta with (0,0) and camera (FreeFloatMovement) does not rotate. Issue is also on dev. Tested in Standalone.

Please checkout branches and confirm the issue.

Diffed with dev branch, seems that there are no changes in this branch for regression.
![image](https://user-images.githubusercontent.com/5374481/48089709-67dc3a00-e205-11e8-88c7-79becde21be2.png)

Reply:
Indeed there is a problem, this is a result the way event are being sent. MOUSEMOVE and MOUSEPOS are always received in a pair (with that order). Let's add a comment describing that behaviour, and remove the commended line of code.

Reply:
Done